### ISSUE TITLE
wrong Parsing for new Object({})

### ISSUE THREAD

**Issue by Vintic** [cflint/CFLint#669]
Tested for UNUSED_LOCAL_VARIABLE.
LOCAL.myVar - is reported as unused.

```cfml
<cfcomponent>
    <cffunction name="actionList">
        <cfset LOCAL.myVar = test(
            argName = "test"
        )>
        <cfset  LOCAL.secondVar = new myObj({
            varName = LOCAL.myVar
        })>
    </cffunction>
</cfcomponent>
```

**Comment by KamasamaK** [comment]
@Vintic Can you check again with the latest build? I think c4006c9 may have fixed this.

**Comment by Vintic** [comment]
> @Vintic Can you check again with the latest build? I think [c4006c9](https://github.com/cflint/CFLint/commit/c4006c96f1f3a757f9c3f20a46c231a2aceb2c43) may have fixed this.

Yes, fixed, thanks you. Should I close issue?

**Comment by KamasamaK** [comment]
I don't see why we would leave the issue opened if it's resolved. Also, duplicate of #640. I'll close it.

**Comment by ryaneberly** [comment]
Thanks @KamasamaK
